### ISSUE TITLE
precision and recall equal 0, two mAPs are very small

### ISSUE THREAD

**Issue by PALMJJ** [ultralytics/yolov5#520]
I train my own face mask dataset. There are only two kinds of data sets: wearing masks and not wearing masks. There are 400 training sets and 100 validation sets. Why does this happen during training? Hope to get a reply. I compared the training process you gave.
<img width="946" alt="1" src="https://user-images.githubusercontent.com/42363091/88471703-c8ae2780-cf3e-11ea-9174-e4483b8d2d35.PNG">

**Comment by glenn-jocher** [comment]
@PALMJJ do not use a bug label for questions. What are you asking exactly?
